pkg/components/exporter/exporter_test.go

```diff
@@ -17,6 +28,25 @@ func TolerationsFromDS(ds *appsv1.DaemonSet) []corev1.Toleration {
 	return ds.Spec.Template.Spec.Tolerations
 }
 
+func HostPIDFromDS(ds *appsv1.DaemonSet) bool {
+	return ds.Spec.Template.Spec.HostPID
+}
+
+func VolumeMountsFromDS(ds *appsv1.DaemonSet) []corev1.VolumeMount {
+	return ds.Spec.Template.Spec.Containers[0].VolumeMounts
+}
+
+func AllowsFromSCC(SCC *secv1.SecurityContextConstraints) SCCAllows {
+	return SCCAllows{
+		AllowPrivilegedContainer: SCC.AllowPrivilegedContainer,
+		AllowHostDirVolumePlugin: SCC.AllowHostDirVolumePlugin,
+		AllowHostIPC:             SCC.AllowHostIPC,
+		AllowHostNetwork:         SCC.AllowHostNetwork,
+		AllowHostPID:             SCC.AllowHostPID,
+		AllowHostPorts:           SCC.AllowHostPorts,
+	}
+}
```

Comment:
nit: (feel free to ignore): I think these generic utility functions are better placed in pkg/utils/k8s so that these are useful in e2e as and in controllers as well.

Reply:
+1